src/server.js

```diff
@@ -8,11 +8,15 @@ import oauthRoutes from './routes/oauth.js';
 import studentsRoutes from './routes/students.js';
 import usersRoutes from './routes/users.js';
 
+import { formattedSendStatus } from './utils/sendStatusJSONFormatter.js';
+
 const app = express();
 
 app.use(cors());
 app.use(express.json());
 
+app.response.sendStatus = formattedSendStatus;
```

Comment:
Só deixa essa linha logo após a criação do `app`, já que ela altera uma função interna da lib.
```suggestion
const app = express();
app.response.sendStatus = formattedSendStatus;

app.use(cors());
app.use(express.json());
```